index.js

```diff
@@ -6,58 +6,30 @@ export default function pLimit(concurrency) {
 	const queue = new Queue();
 	let activeCount = 0;
 
-	const resumeNext = () => {
+	const next = () => {
 		if (activeCount < concurrency && queue.size > 0) {
-			queue.dequeue()();
-			// Since `pendingCount` has been decreased by one, increase `activeCount` by one.
 			activeCount++;
+			queue.dequeue()();
```

Comment:
Is there any reason to change the dequeue timing and/or remove the comment?

If not, could you revert back to the original? It is easier to understand the purpose of this PullRequest if you keep it as it was.

If there is a reason, it should be noted in the comments.